package/obs/rmt-server.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Jun  6 13:29:12 UTC 2019 - Leon Schroeder <lschroeder@suse.com>
+
+- Version 2.1.5
+- Fix create UUID when system_uuid file empty
```

Comment:
@s-leon @tmuntaner The changelog entry needs a `bsc` bug number -- otherwise it will be rejected by maintenance.

Reply:
I had a quick search of bugzilla and couldn't find anything. Made bsc#1138316 instead.